noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/types/token_note.nr

```diff
@@ -157,7 +157,7 @@ impl OwnedNote for TokenNote {
  *
  * However we can still perform addition/subtraction on points! That is why we generate those two points, which are:
  * incomplete_fee_payer_point := G_npk * fee_payer_npk + G_rnd * fee_payer_randomness
- * incomplete_user_point := G_npk * user_npk + G_amt * funded_amount + G_rnd * user_randomness
```

Comment:
I imagine that `x + G_amt * y` is equal to `x` if `y == 0`? So removing this term is the same as saying `amount = 0`

Reply:
I don't follow what you are trying to say here but I've noticed that the related comments needed to be updated as well and I did that in https://github.com/AztecProtocol/aztec-packages/pull/7648/commits/c0570e65ddd3a4288ab62ade89f33d7c127b7324

Reply:
I meant that the comment now doesn't list `amount` as being included in the computation, I was thinking how this is equivalent to `amount` actually being there, but being equal to 0.

This is relevant for things like #7585, since we'd want to avoid having to track whether the amount is inside the point or not.